src/ProofOfConcepts/connectors/connectors.tsx

```diff
@@ -0,0 +1,363 @@
+import React from "react";
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Flex,
+  FlexItem,
+  Label,
+  LabelGroup,
+} from "@patternfly/react-core";
+import PlusCircleIcon from "@patternfly/react-icons/dist/esm/icons/plus-circle-icon";
+import OutlinedQuestionCircleIcon from "@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon";
+
+export const DataListSelectableRows: React.FunctionComponent = () => {
```

Comment:
It's a good habit to keep the filename consistent with the component name.  Maybe going forward once the components start to feel more specific you might use a name like `ConnectorTypeListItem` for example.